frontend/src/lib/lemon-ui/LemonTable/columnUtils.tsx

```diff
@@ -42,3 +42,21 @@ export function createdByColumn<T extends { created_by?: UserBasicType | null }>
             ),
     }
 }
+
+export function updatedAtColumn<T extends { updated_at?: string | Dayjs | null }>(): LemonTableColumn<T, 'updated_at'> {
+    return {
+        title: 'Updated',
+        dataIndex: 'updated_at',
+        render: function RenderCreatedAt(created_at) {
+            return created_at ? (
+                <div className="whitespace-nowrap text-right">
+                    <TZLabel time={created_at} />
```

Comment:
```suggestion
        render: function RenderCreatedAt(updated_at) {
            return updated_at ? (
                <div className="whitespace-nowrap text-right">
                    <TZLabel time={updated_at} />
```
Seems to be a copy-and-paste oversight?